podcasts/End of Year/Stories/EpilogueStory.swift

```diff
@@ -24,19 +29,16 @@ struct EpilogueStory: StoryView {
 
                 StoryLabelContainer(topPadding: 0, geometry: geometry) {
                     if visibility.isVisible {
-                        HolographicEffect(parentSize: geometry.size) {
+                        GradientHolographicEffect(parentSize: geometry.size) {
```

Comment:
I created a `GradientHolographicEffect` that rotates the gradient. However, the effect is so subtle that I'm considering removing it at all.

Reply:
I can see that, I played with some ideas but didn't come up with anything better yet. We can remove it for now and just do a gradient overlay.